### ISSUE TITLE
Error in RNNModel - LSTM after specifying lr_scheduler_cls

### ISSUE THREAD

**Issue by shashankdeshpande** [unit8co/darts#707]
Getting error while fitting RNNModel-LSTM if we specify lr_scheduler_cls

**To Reproduce**
Code snippet
```python
lstm_model = RNNModel(
	model='LSTM',
	optimizer_kwargs={'lr': 1e-3},
	lr_scheduler_cls=torch.optim.lr_scheduler.ReduceLROnPlateau,
	...
	)
```
Error Screenshot
![Screenshot (35)](https://user-images.githubusercontent.com/47742503/147477737-42380f03-5a63-41b5-b57d-286616674786.png)

**System**
 - Python version: 3.8.12
 - darts version: 0.14.0
 - torch version: 1.10.1+cpu

**Comment by pennfranc** [comment]
Thanks for reporting! I added support for `ReduceLROnPlateau` in #711, at least using the training loss as a metric.
